doc/manual/team.rst

```diff
@@ -225,6 +225,9 @@ TOO-LATE
 
 The judges may have prepared multiple test files for each problem.
 DOMjudge will report back the first non-correct result as verdict.
+*DOMjudge will always output the verdict on the first ordered testcase.
+This means that when the submission is deterministic DOMjudge will give back
+the same output on resubmission.*
```

Comment:
Why is this important to know for the team and why does it need to be made so explicit?

Reply:
I've had teams submitting multiple times in the hope to get another outcome, now that we have parallel judging a team could think that they can have more information with a small penalty.

If I would know that I only have WA I might fix it whereas if I also got TLE I might skip this problem and focus on another one.

Reply:
Not sure the manual is the place to protect people from penalty while trying to game the system

Reply:
Ok, I'll rip it out and only address @nickygerritsen his comment.